modules/rest-http/src/main/java/org/apache/ignite/internal/processors/rest/protocols/http/jetty/GridJettyRestProtocol.java

```diff
@@ -123,10 +123,10 @@ public GridJettyRestProtocol(GridKernalContext ctx) {
         InetAddress locHost;
 
         try {
-            locHost = U.resolveLocalHost(ctx.config().getLocalHost());
+            locHost = U.resolveLocalHost(System.getProperty(IGNITE_JETTY_HOST, ctx.config().getLocalHost()));
```

Comment:
I think it is better to rename "locHost" to "jettyHost" now.

Reply:
Got it.